etc/rucio.cfg.atlas.client.template

```diff
@@ -14,4 +15,6 @@ rucio_host = https://voatlasrucio-server-prod.cern.ch:443
 auth_host = https://voatlasrucio-auth-prod.cern.ch:443
 ca_cert = $RUCIO_HOME/etc/ca.crt
 client_x509_proxy = $X509_USER_PROXY
-request_retries = 3
\ No newline at end of file
+request_retries = 3
+support_experiment = hn-atlas-dist-analysis-help@cern.ch
```

Comment:
I think `support_vo` would be a better name and more in line what we have now with the policy/permission as well.

Reply:
organyzation_support ?

Reply:
organization_support sounds good to me. But we should change this consistently throughout all files, so VO and organization is not mixed.